calico.go

```diff
@@ -191,7 +191,13 @@ func cmdAdd(args *skel.CmdArgs) error {
 			return err
 		}
 	} else {
-		// Default CNI behavior - use the CNI network name as the Calico profile.
+		// Default CNI behavior
+		// Validate enabled features
+		if conf.FeatureControl.IPAddrsNoIpam {
+			return errors.New("requested feature is not supported for this orchestrator: ip_addrs_no_ipam")
```

Comment:
I think we should swap out the word "orchestrator" for "runtime" or "environment" or something. It's possible to have a Kubernetes deployment using a different runtime that doesn't support this feature, for example.